com.cerebralnomad.recipescribe.yml

```diff
@@ -0,0 +1,25 @@
+id: com.cerebralnomad.recipescribe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: recipe_scribe-2.0.1
+modules:
+  - name: recipescribe
+    buildsystem: simple
+    build-commands:
+      - rm README.txt
```

Comment:
```suggestion
```

Reply:
I didn't know if that was needed or not. The tarball is in my releases and i only included the readme in case it was downloaded by mistake instead o fthe proper tarball of the standalone program.
I'll remove that line